Sources/Public/Animation/LottieView.swift

```diff
@@ -115,26 +115,23 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
   // MARK: Public
 
   public var body: some View {
-    ZStack {
-      if let animationSource {
-        LottieAnimationView.swiftUIView {
-          defer { animationDidLoad?(animationSource) }
-          return LottieAnimationView(
-            animationSource: animationSource,
-            imageProvider: imageProviderConfiguration?.imageProvider,
-            textProvider: textProvider,
-            fontProvider: fontProvider,
-            configuration: configuration,
-            logger: logger)
-        }
-        .sizing(sizing)
-        .configure { context in
-          applyCurrentAnimationConfiguration(to: context.view)
-        }
-        .configurations(configurations)
-      } else {
-        placeholder?()
-      }
+    LottieAnimationView.swiftUIView {
+      LottieAnimationView(
+        animationSource: animationSource,
+        imageProvider: imageProviderConfiguration?.imageProvider,
+        textProvider: textProvider,
+        fontProvider: fontProvider,
+        configuration: configuration,
+        logger: logger)
+    }
+    .sizing(sizing)
+    .configure { context in
+      applyCurrentAnimationConfiguration(to: context.view)
+    }
+    .configurations(configurations)
+    .overlay {
+      placeholder?()
+        .opacity(animationSource == nil ? 1 : 0)
```

Comment:
Should we also set `.opacity(animationSource == nil ? 0 : 1)` on the `LottieAnimationView`? That may best match the existing behavior of an `if` where we only every displayed one or the other

Reply:
Yes, I think this will be necessary -- if you change `.reloadAnimationTrigger(currentURLIndex, showPlaceholder: false)` to `showPlaceholder: true` in `AnimationPreviewView` and test the "Multiple Animations" demo, you'll see the behavior difference:

| Before | With this current change |
| ---- | ---- |
| <img width="731" alt="Screenshot 2024-01-19 at 4 29 11 PM" src="https://github.com/airbnb/lottie-ios/assets/1811727/4444c7bf-cdc3-4bb6-9103-56697b66a3fb"> | <img width="731" alt="Screenshot 2024-01-19 at 4 30 03 PM" src="https://github.com/airbnb/lottie-ios/assets/1811727/8d29faa7-161b-4e44-abc7-779282a68fa9"> |